tests/phpunit/includes/Core/Modules/FakeModule.php

```diff
@@ -136,6 +142,11 @@ protected function create_data_request( Data_Request $data ) {
 					$data = $data->data;
 					return json_encode( compact( 'method', 'datapoint', 'data' ) );
 				};
+			case 'POST:test-request':
```

Comment:
The handler is the same as GET in which case the cases can be combined using fall-through.

Reply:
Ah sorry, it's when multiple `cases` are handled in the same way like so:
https://github.com/google/site-kit-wp/blob/878d7fd21a61b6a1110d4dd080a8d7ce2de5a5ec/includes/Modules/Tag_Manager.php#L259-L262

Reply:
Updated, thank you!